catalog-rest-service/src/main/resources/json/schema/entity/services/databaseService.json

```diff
@@ -110,6 +112,12 @@
         },
         {
           "name": "SampleData"
+        },
+        {
+          "name": "Atlas"
```

Comment:
why are we adding this here?

Reply:
To create database service we need to add respective name in json, otherwise it throws error.

Reply:
I think the question was more related towards why are we placing it as a databaseService. I believe it should go in `metadataService` https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/entity/services/metadataService.json

Reply:
I think this would require us to add the metadataService APIs on the backend